react-backend/app.js

```diff
@@ -45,39 +16,8 @@ app.use(bodyParser.urlencoded({ extended: true }));
 app.use(cookieParser());
 app.use(express.static(path.join(__dirname, 'client/build')));
 
-// Open routes
-app.use('/cb/register', cbRegister);
-app.use('/cb/register/check', cbRegisterCheck);
-app.use('/cb/login', cbLogin);
-app.use('/cb/pwd/change', cbPasswordChange);
-app.use('/cb/pwd/reset', cbPasswordReset);
-
-// Authenticated routes
-app.use('/qr/generator', mwIsAuthenticated, qrGenerator);
-app.use('/user/name-from-scan', mwIsAuthenticated, visitorName);
-app.use('/visit/check', mwIsAuthenticated, visitorCheck);
-app.use('/visit/add', mwIsAuthenticated, visitInsert);
-app.use('/activities/today', mwIsAuthenticated, activitiesToday);
-app.use('/admin/login', mwIsAuthenticated, adminLogin);
-
-// Admin routes
-app.use('/user/qr/email', mwAdminIsAuthenticated, qrSend);
-app.use('/user/qr', mwAdminIsAuthenticated, qrUserGen);
-app.use('/cb/details', mwAdminIsAuthenticated, cbDetails);
-app.use('/cb/details/update', mwAdminIsAuthenticated, cbDetailsNew);
-app.use('/user/details', mwAdminIsAuthenticated, userDetails);
-app.use('/user/details/update', mwAdminIsAuthenticated, userDetailsNew);
-app.use('/users/all', mwAdminIsAuthenticated, usersAll);
-app.use('/users/chart-all', mwAdminIsAuthenticated, usersChartAll);
-app.use('/users/filtered', mwAdminIsAuthenticated, usersFiltered);
-app.use('/visitors/all', mwAdminIsAuthenticated, visitorsAll);
-app.use('/visitors/filtered', mwAdminIsAuthenticated, visitorsFiltered);
-app.use('/admin/check', mwAdminIsAuthenticated, adminCheck);
-app.use('/activities/all', mwAdminIsAuthenticated, activitiesAll);
-app.use('/activity/add', mwAdminIsAuthenticated, activitiesAdd);
-app.use('/activity/update', mwAdminIsAuthenticated, activitiesUpdate);
-app.use('/activity/delete', mwAdminIsAuthenticated, activitiesDelete);
-
+router.use('/api', routes);
+app.use(router);
```

Comment:
I think you're creating an unnecessary router here. Can't you just do `app.use('/api', routes)`?

Also probably better to call it `apiRoutes` or something like that.